client/src/pages/organizations/Form.js

```diff
@@ -344,6 +360,18 @@ class BaseOrganizationForm extends Component {
 		arrayHelpers.push({name: '', approvers: []})
 	}
 
+	removeApprovalStep = (arrayHelpers, index, step) => {
+		return API.query(/* GraphQL */`
+			approvalStepInUse(uuid:"${step.uuid}") 
+		`).then(data => {
+			if (data) {
+				this.setState({showRemoveApprovalStepAlert: true})
+				return
```

Comment:
In this case it is probably slightly neater to use `} else { … }` instead of a return (although that is mostly a matter of taste).

Reply:
As you said, it is a matter of taste. I do prefer to short-circuit when certain conditions are not met, but since it is a really simple statement, an if-else also clearly conveys the intent. I will change it.